pickee_api/models.py

```diff
@@ -12,85 +13,94 @@ class UserProfile(models.Model):
     avatar = models.CharField(max_length=500)
 
     GENDER_CHOICES = [
-        ('MALE', 'Male'), 
-        ('FEMALE', 'Female'), 
-        ('OTHER','Other'), 
-        ('UNSPECIFIED','Prefer not to say')]
+        ('MALE', 'Male'),
+        ('FEMALE', 'Female'),
+        ('OTHER', 'Other'),
+        ('UNSPECIFIED', 'Prefer not to say')]
     gender = models.CharField(max_length=18, choices=GENDER_CHOICES)
     age = models.IntegerField(validators=[MaxValueValidator(100), MinValueValidator(0)])
-    associated_users = models.ManyToManyField('self')
+    associated_users = models.ManyToManyField('self', blank=True)
 
     def __str__(self):
         return self.user.first_name + " " + self.user.last_name
 
+
 class FavoriteActor(models.Model):
     user = models.ForeignKey('UserProfile', on_delete=models.CASCADE)
     actor = models.ForeignKey('Actor', on_delete=models.CASCADE)
 
     def __str__(self):
         return self.actor.name
 
+
 class Actor(models.Model):
-    person_id = models.IntegerField()
+    id = models.IntegerField(primary_key=True)
     name = models.CharField(max_length=128)
-    picture = models.ImageField()
+    image_url = models.URLField(blank=True, null=True)
 
     def __str__(self):
         return self.name
 
+
 class FavoriteMovie(models.Model):
     user = models.ForeignKey('UserProfile', on_delete=models.CASCADE)
     movie = models.ForeignKey('Movie', on_delete=models.CASCADE)
 
     def __str__(self):
         return self.user + ": " + self.movie
 
+
 class Movie(models.Model):
-    movie_id = models.IntegerField()
+    id = models.IntegerField(primary_key=True)
     name = models.CharField(max_length=128)
-    picture = models.ImageField()
-    rating = models.IntegerField(validators=[MaxValueValidator(100), MinValueValidator(0)])
+    image_url = models.URLField(blank=True, null=True)
+    rating = models.DecimalField(max_digits=2, decimal_places=1, validators=[MaxValueValidator(10.0), MinValueValidator(0)])
```

Comment:
Not sure how it works, but will it allow rating 100? Because there are movies with such meta score (https://www.metacritic.com/movie/the-godfather)

Reply:
@Arden488 it depends on what kind of data we want to store in `rating`. After taking a look at tmdb response, I spotted that `vote_average` held a decimal value. Thus, this change above. We can change to whatever we prefer because we haven't deployed anything yet.

https://api.themoviedb.org/3/movie/popular